suttas/translation_en/mn/mn10_translation-en-anigha.json

```diff
@@ -33,7 +33,7 @@
   "mn10:5.4": "That’s how a bhikkhu abides maintaining perspective of the body concurrently with the body. ",
   "mn10:5.5": " ",
   "mn10:6.0": "1.2. The Postures ",
-  "mn10:6.1": "Furthermore, when a bhikkhu is walking he understands: ‘I am walking.’ When standing he understands: ‘I am standing.’ When sitting he understands: ‘I am sitting.’ And when lying down he understands: ‘I am lying down.’ ",
+  "mn10:6.1": "Furthermore, when a bhikkhu is walking he understands: ‘I am walking.’ When standing he understands: ‘I am standing.’ When sitting he understands: ‘I am sitting.’ And when lying down he understands: ‘I am lying down.’? ",
```

Comment:
Looks like a typo?

Reply:
Oh you're right, my bad, it is fixed now.